openstack/containerinfra/v1/clustertemplates/testing/fixtures.go

```diff
@@ -232,7 +232,6 @@ var ExpectedClusterTemplate = clustertemplates.ClusterTemplate{
 }
 
 var ExpectedClusterTemplate_EmptyTime = clustertemplates.ClusterTemplate{
-	APIServerPort:       "",
```

Comment:
Would it be better to have it as APIServerPort: 0?

Reply:
six one way, half a dozen the other, I think.

Having it set to 0 would be more explicit, but both ways are valid and work.